jetty-util/src/main/java/org/eclipse/jetty/util/security/Constraint.java

```diff
@@ -37,6 +37,8 @@
     public static final String __SPNEGO_AUTH = "SPNEGO";
     public static final String __NEGOTIATE_AUTH = "NEGOTIATE";
 
+    public static final String __OPENID_AUTH = "OPENID";
```

Comment:
Why there is an empty like before __OPENID_AUTH?

Reply:
removed empty lines